front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts

```diff
@@ -63,6 +63,11 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
     validateService.formValidatorSchema = transactionType?.schema;
     validateService.formValidatorForm = form;
 
+    // Override contact type options if present in transactionType
+    if (transactionType && transactionType.contactTypeOptions) {
+      this.contactTypeOptions = this.getContactTypeOptions(transactionType.contactTypeOptions);
```

Comment:
is this essentially equivalent to:
```
    if (transactionType && transactionType.contactTypeOptions) {
      this.contactTypeOptions = transactionType.contactTypeOptions.map((type: ContactTypes) => ({
        name: LabelUtils.get(ContactTypeLabels, type),
        code: type,
      }));
    }

Reply:
I could see us using this pattern elsewhere when we want a PrimeOptions list given a list of 'code's and one of our homebrew LabelLists

maybe in LabelUtils we could add

```

  public static getPrimeOptions(labelArrays: LabelList, options: string[]) {
    return options.map((type: string) => ({
      name: LabelUtils.get(labelArrays, type),
      code: type,
    }));
  }

Reply:
I agree that the getContactTypeOptions() code would be good to generalize out into the LabelUtils class.

Reply:
I like this idea a lot.  I'll take a look at it!

Reply:
https://github.com/fecgov/fecfile-web-app/pull/813/files#diff-5bf0fd4e8831d3a8ff3905e7c9a0a450bbeb72fb77c6cd9c7438a604a6925517R55

LabelUtil's getPrimeOptions was updated to allow you to optionally specify the values to use.